### PR TITLE
Add invalid switch case matcher

### PR DESCRIPTION
**Pull request by brianyu28**
Addressing #73, and `/review` inputs 1934, 1937, and 1941.

### CODE REVIEW

File: helpers/clang.py [hunk 1]
```diff
@@ -281,6 +281,19 @@ def help(lines):
...
+    # foo.c:7:15: error: expression is not an integer constant expression
+    #         case (x > 28):
+    #              ~^~~~~~~
+    matches = re.search(r"^([^:]+):(\d+):\d+: error: expression is not an integer constant expression", lines[0])
+    if matches:
+        response = [
+            "Remember that each `case` in a `switch` statement needs to be an integer (or a `char`, which can be represented as an integer).",
+            "You aren't allowed to include other types or boolean expressions in a `switch` `case`."
+        ]
+        return (2, response) if len(lines) >= 2 else (1, response)
```
Comment (thctamm) [inline helpers/clang.py]: If you're only matching on the first line, then why ever return (2, response)?
just return (1, response) seems more appropriate.

**Comment by thctamm** [comment]
Looks good to me

File: helpers/clang.py [hunk 2]
```diff
@@ -281,6 +281,19 @@ def help(lines):
...
     # $ clang foo.c
     # /tmp/foo-1ce1b9.o: In function `main':
+    # foo.c:7:15: error: expression is not an integer constant expression
+    #         case (x > 28):
+    #              ~^~~~~~~
+    matches = re.search(r"^([^:]+):(\d+):\d+: error: expression is not an integer constant expression", lines[0])
+    if matches:
+        response = [
+            "Remember that each `case` in a `switch` statement needs to be an integer (or a `char`, which can be represented as an integer).",
+            "You aren't allowed to include other types or boolean expressions in a `switch` `case`."
```
Comment (dmalan) [inline helpers/clang.py]: `, not a Boolean expression or other type`

File: helpers/clang.py [hunk 3]
```diff
@@ -287,8 +287,7 @@ def help(lines):
     matches = re.search(r"^([^:]+):(\d+):\d+: error: expression is not an integer constant expression", lines[0])
     if matches:
         response = [
-            "Remember that each `case` in a `switch` statement needs to be an integer (or a `char`, which can be represented as an integer).",
-            "You aren't allowed to include other types or boolean expressions in a `switch` `case`."
+            "Remember that each `case` in a `switch` statement needs to be an integer (or a `char`, which can be represented as an integer), not a Boolean expression or other type."
```
Comment (dmalan) [inline helpers/clang.py]: "which is really just an integer"

**Comment by brianyu28** [comment]
Fixed!

**Pull request merged**